codechecker_common/skiplist_handler.py

```diff
@@ -33,6 +33,8 @@ def __init__(self, skip_file_content=""):
         Process the lines of the skip file.
         """
         self.__skip = []
+        if not skip_file_content:
+            skip_file_content = ""
```

Comment:
Do we need this? By default it is an empty string and also a string should be given if this parameter is specified.

Reply:
The constructor can be called:

- SkipListHandler("blabla")
- SkipListHandler("")
- SkipListHandler()
- SkipListHandler(None)

This supports the latest form of invocation.

Reply:
By the type system the parameter `skip_file_content` is a string, so you can't specify a None value to this variable. You have to you Optional[str] if you want to allow None value for a parameter.